src/Controllers/DeleteEntryController.php

```diff
@@ -21,9 +21,33 @@ public function __construct($BlogModel)
 
     public function __invoke(Request $request, Response $response, array $args)
     {
-        $DeletedEntry = $request->getParsedBody();
-        $DbResult = $this->BlogModel->DeleteEntry($DeletedEntry);
-        return $response->withHeader('Location', '/');
+        $ResponseData =
+            [
+                'success' => false,
+                'message' => '',
+                'data' => []
+            ];
+
+        $BlogPost = new Valitron\Validator($_POST);
+        $BlogPost->rule('required', ['GUID']);
+
+        if ($BlogPost->validate()) {
+            $Result = $this->BlogModel->DeleteEntry($DeletedEntry);

```

Comment:
what is $DeletedEntry?

Reply:
i assume its meant to be the GUID for the post but that's not clear, and also it doesn't exist anymore

Reply:
good catch. thanks